portfolio/src/main/webapp/index.html

```diff
@@ -22,11 +22,36 @@
       </div>
       
       <section id="intro">
-        <h1 class="big-header-main">Hello, I'm <span style="color: #9366e3">Sabrina Snider</span>.</h1>
-        <h2 class="sub-header-main">I like writing code.</h2>
-        <a href="#work-section" id="work-button" class="white-button">My Work</a>
+        <div id="intro-content">
+          <div>
+            <div id="intro-text">
+              <h2 class="sub-header-main">Hello World</h2>
+              <h1 class="big-header-main">I'm <span style="color: #9366e3; font-weight: bold;">Sabrina</span> and I like to code</h1>
+              <a id="portfolio-button" class="white-button" href="#about-me-section"
+                onmouseover="portfolioBtnHoverOn()"
+                onmouseout="portfolioBtnHoverOff()"
+              >
+                View my portfolio
+                <img id="portfolio-arrow" src="static-files/images/icons/arrow.svg" >
+              </a>
+            </div>
+          </div>
+
+          <div id="intro-photo-container">
+            <img id="intro-photo" src="./static-files/images/selfie-cropped.jpg">
+          </div>
+        </div>
+
+        <div class="circle circle-small"><img id="circle-image-small" src="static-files/images/circle.svg"></div>
+        <div class="circle circle-large"><img id="circle-image-large" src="static-files/images/circle2.svg"></div>
       </section>
 
+      <div id="overlay">
+        <div class="layer layer-1"></div>
+        <div class="layer layer-2"></div>
+        <div class="layer layer-3"></div>
```

Comment:
When are you populating ```layer-1```, ```layer-2```, ```layer-3```?

Also, I'm trying to figure out what each layer is used for. Would there be other names you could use the differentiate them?